htmlmin/command.py

```diff
@@ -152,8 +150,7 @@ def main():
   if args.input_file:
     inp = codecs.open(args.input_file, encoding=args.encoding)
   else:
-    inp = codecs.getreader(
-      sys.stdin.encoding or locale.getpreferredencoding())(sys.stdin)
+    inp = io.open(0, encoding=args.encoding)
```

Comment:
Instead of using args.encoding universally, we should use sys.stdin.encoding as a fallback if args.encoding was not set. This means settings args.encoding to `None` by default and then checking if it is `None` at the top of main:

    encoding_set = args.encoding is None or args.encoding == ''
    args.encoding = 'utf-8' if not encoding_set else args.encoding